akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -63,18 +63,18 @@ what clients are connected.
 Response messages from the destination actor are tunneled via the receptionist
 to avoid inbound connections from other cluster nodes to the client:
 
-* @scala[`sender()`,] @java[`getSender()`,] as seen by the destination actor, is not the client itself, 
+* @scala[@scaladoc[`sender()`,](akka.actor.Actor)]@java[@javadoc[`getSender()`,](akka.actor.Actor)] as seen by the destination actor, is not the client itself,
   but the receptionist
-* @scala[`sender()`] @java[`getSender()`] of the response messages, sent back from the destination and seen by the client, 
+* @scala[@scaladoc[`sender()`](akka.actor.Actor)] @java[@javadoc[`getSender()`](akka.actor.Actor)] of the response messages, sent back from the destination and seen by the client,
```

Comment:
Similar to [this comment](https://github.com/akka/akka/pull/24524/files#r167429853), the URL for the `sender()` Scala method includes the return type:
https://doc.akka.io/api/akka/2.5/akka/actor/Actor.html#self:akka.actor.ActorRef

So I avoid this messy URL and just link it to the `Actor` trait Scaladoc.